### PR TITLE
JBPM-7477 - New Docker image getting started

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: learn/gettingStarted.html.haml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      or
+
+        $ docker ps -a
+
+      In case you want to also list stopped containers.
+
+      ==== Stopping the jBPM Server Full Distribution container
+        $ docker stop jbpm-server-full
+
+      ==== Delete jBPM Server Full Distribution container and all itś data
```
Comment (mswiderski) [inline learn/gettingStarted.html.haml]: I see you started to use polish character :)

> itś

File: download/download.html.haml [hunk 2]
```diff
@@ -117,6 +117,22 @@ title: Download jBPM
 
       * http://registry.hub.docker.com/u/jboss/jbpm-workbench/[jBPM Workbench]
       * http://registry.hub.docker.com/u/jboss/jbpm-workbench-showcase/[jBPM Workbench Showcase]
-      * http://registry.hub.docker.com/u/jboss/jbpm-server-full/[jBPM Server Full (zip)]
+      * http://registry.hub.docker.com/u/jboss/jbpm-server-full/[jBPM Server Full]
```
Comment (mswiderski) [inline download/download.html.haml]: I'd actually put the server full as first one and the other two below (even in separate section if at all) - why we have two workbenches and no KIE Server?

File: learn/gettingStarted.html.haml (hunk 1 above)
Comment (cristianonicolai) [reply learn/gettingStarted.html.haml]: trying to get some points for my PR :P

File: download/download.html.haml (hunk 2 above)
Comment (cristianonicolai) [reply download/download.html.haml]: Sure, changed the order and added link to kie server image.

**Comment by cristianonicolai** [comment]
@mswiderski I think it can be merged now, the image is available already. The release notes file will only be available when we update https://github.com/kiegroup/jbpm-website/blob/master/learn/releases.html.haml#L24

**Comment by mswiderski** [comment]
@cristianonicolai and what about wbadmin user? I guess it's not working with 7.9.0, isn't it?

**Comment by cristianonicolai** [comment]
@mswiderski that's true, it works as admin/admin now... I can fix the instructions and open a separated PR for that or we just wait to merge this.

**Comment by mswiderski** [comment]
either way is fine but I'd prefer to avoid situation when the first thing people will try will fail - use of wbadmin... so we can either publish the changes about docker server full now but with same information as we have for the single zip and then updates with wbadmin at 7.10 release

**Comment by cristianonicolai** [comment]
definitely, it needs to be consistent.
So, I updated the user reference to _admin_ where applicable and will open a new PR to update back to _wbadmin_ that will have to wait until we release 7.10.

**Pull request merged**